packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerScenario.ts

```diff
@@ -99,10 +116,24 @@ interface ScenarioBuilderState {
 type TestCommit = Commit<TestChange> & {
 	seqNumber: SeqNumber;
 	refNumber: SeqNumber;
+	intention: number;
 };
 
 const localSessionId: SessionId = "0" as SessionId;
 
+/**
+ * Get the first push step that for which an ack step has not been generated.
+ * @param scenario - The scenario with all the steps.
+ * @param inflight - The number of steps that are inflight, i.e., have not been acked yet.
+ * @returns the first push step that for which an ack step has not been generated.
+ */
+function getFirstUnackedStep(scenario: readonly UnitTestScenarioStep[], inflight: number) {
```

Comment:
Looks like this code is no longer needed?

Reply:
oh yeah, I used this in an earlier iteration.